cristin-import/src/main/java/no/unit/nva/cristin/lambda/CristinEntryEventConsumer.java

```diff
@@ -2,13 +2,15 @@
 
 import static no.unit.nva.cristin.lambda.constants.HardcodedValues.HARDCODED_PUBLICATIONS_OWNER;
 import static no.unit.nva.publication.s3imports.ApplicationConstants.MAX_SLEEP_TIME;
+import static no.unit.nva.publication.s3imports.FileEntriesEventEmitter.timestampToString;
```

Comment:
Utility methods should be gathered together in independent classes. If we ever get rid of the FileEntriesEventEmitter we would have to change all classes that are importing this method but have otherwise nothing to do with the class.